app/controllers/api/base_controller/parameters.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   class BaseController
     module Parameters
+      include CompressedIds
```

Comment:
@imtayadeway just tried adding it to the top `BaseController` in my issue to see if it made a difference, but it didn't 😞